src/tests/data/BAV_CRI_AUTH_CODE_ISSUED_SCHEMA.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Schema looks good, but if we're only validating 1 of the 2 TxMA events returned (BAV_CRI_START and BAV_CRI_AUTH_CODE_ISSUED) are you sure this schema is being used?